elftools/dwarf/dwarf_expr.py

```diff
@@ -169,6 +175,29 @@ def _visit_OP_addr(self, opcode, opcode_name):
         self._cur_args = [
                 struct_parse(self.structs.Dwarf_target_addr(''), self.stream)]
 
+    # ULEB128, then an expression of that length
+    def _visit_OP_nestedexpr(self, opcode, opcode_name):
+        size = struct_parse(self.structs.Dwarf_uleb128(''), self.stream)
+        nested_expr_blob = self.stream.read(size)
+        if isinstance(nested_expr_blob, str): # Python 2 patch. In Py3, it comes as bytes.
+            nested_expr_blob = [ord(b) for b in nested_expr_blob]
+        dumper = GenericExprDumper(self.structs)
+        # If used from a dumper context, reuse the outer operation formatter
```

Comment:
Ugh, this seems extremely hacky, sorry. The circular dependency between visitor and dumper, the magic discovery of what "context" you're used in. I wish there was a cleaner way to design this. DWARF has about 17000 dark corners and gotchas, if we implement each with a cut-through-all-layers hack, pyelftools will become completely unusable very soon. I'd rather we not support this opcode for now if there's no better solution.